apps/backoffice-v2/src/domains/customer/fetchers.ts

```diff
@@ -39,12 +39,14 @@ const CustomerSchema = z.object({
       isExample: z.boolean().default(false),
       isDemo: z.boolean().default(false),
       showFullAccessPopup: z.boolean().default(false),
+      demoAccount: z.boolean().default(false),
```

Comment:
`is` prefix will make the flag more consistent with the other flags we have

```suggestion
      isDemoAccount: z.boolean().default(false),
```